firmware/boron-ins-fsm/clang-format-all.py

```diff
@@ -9,5 +9,5 @@
 
 import os
 
-os.system("find src -iname *.h -o -iname *.cpp -o -iname *.ino | xargs clang-format -i")
-os.system("find test -iname *.h -o -iname *.cpp -o -iname *.ino | xargs clang-format -i")
\ No newline at end of file
+os.system("find src -iname *.h -o -iname *.cpp -o -iname *.ino | xargs clang-format-12 -i")
+os.system("find test -iname *.h -o -iname *.cpp -o -iname *.ino | xargs clang-format-12 -i")
```

Comment:
I specified `-12` in `clang-format-12` to indicate the specific version I used. If I were to remove it and use just `clang-format`, the system would default to the installed version, which might vary across systems. My intention was to ensure consistent code formatting across all future pull requests. Different versions can have distinct formatting rules and features, so specifying a version helps maintain a uniform style. I tested version 12 with Travis, and it proved to be a reliable choice for our project.

Reply:
If you remember from our training with Theresa, she said that there were inconsistencies with the two clang files, so some systems needed to use it with the `-12` and some needed to use it without. Because of this, I don't think we should be changing the file. Also since Theresa was aware of this and continued to leave the files as is in her subsequent PR's, I think we should do the same.

Reply:
I'd actually be OK with it changing in the files to make them consistently what Travis expects. Then some people will always need to change it and others never, rather than having it like it is now where everyone needs to change it in one file but not the other, lol.